Add FAQ entry for deleting IPs from rate limiter (#2932)
Mailu/Mailu: docs/faq.rst

.. _`github project`: https://github.com/Mailu/Mailu/

How to unblock an IP from rate limiter manually?
````````````````````````````````````````````````

To manually unblock an IP from the rate limiter do the following on your CLI:

.. code-block:: bash

  # list the limited networks (this is not the IP, but only the network part according to AUTH_RATELIMIT_IP_V4_MASK
  $ docker compose exec redis redis-cli -n 2 --scan --pattern 'LIMITER/auth-ip/*'

  # remove from rate limiter
  $ IP=8.8.8.8; docker compose exec redis redis-cli -n 2 --scan --pattern "LIMITER/auth-ip/${IP}/*" \
  | xargs -r docker compose exec -T redis redis-cli -n 2 DEL

> Consider to use :ref:`AUTH tokens` for your users. Ratelimiting is exempted for token-based authentication!

Reviewer: ```suggestion
Consider using :ref:`AUTH tokens` for your users. Token-based authentication is exempted from rate limits!
```

We may want to refer to ``AUTH_REQUIRE_TOKENS`` too
